modules/ROOT/partials/configuration/powerpaste_clean_filtered_inline_elements.adoc

```diff
@@ -5,7 +5,7 @@ This option allows for configuration of PowerPaste's *"clean"* paste filters for
 
 The list of inline elements that should be removed on paste can be specified by setting `+powerpaste_clean_filtered_inline_elements+` to a comma-separated string of inline element tag names.
 
-Possible Values : A comma-separated string.
+*Possible values:* A comma-separated string.
```

Comment:
Looks like this needs to be improved in a follow-up JIra to include a *Type:* and be *Default value:* `+[]+` instead